### PR TITLE
Feat: support HECO

### PR DESCRIPTION
**Pull request by TimonPeng**

### CODE REVIEW

File: verify.js [hunk 1]
```diff
@@ -72,13 +72,16 @@ const parseConfig = (config) => {
...
   const bscscanApiKey = config.api_keys && config.api_keys.bscscan
+  const hecoinfoApiKey = config.api_keys && config.api_keys.hecoinfo
   const ftmscanApiKey = config.api_keys && config.api_keys.ftmscan
 
   const apiKey = apiUrl.includes('bscscan') && bscscanApiKey
     ? bscscanApiKey
     : apiUrl.includes('ftmscan') && ftmscanApiKey
       ? ftmscanApiKey
-      : etherscanApiKey
+      : apiUrl.includes('hecoinfo') && ftmscanApiKey
```
Comment (rkalis) [inline verify.js:82]: This should be `&& hecoinfoApiKey`

**Comment by rkalis** [comment]
Just had the one comment, other than that looks good, thanks!

**Comment by TimonPeng** [comment]
@rkalis Thanks, fixed!

**Comment by rkalis** [comment]
Thanks for the contribution!

**Pull request merged**